app/models/user.rb

```diff
@@ -125,4 +125,8 @@ def remove_story_association
     Story.where(owned_by_id: id).update_all(owned_by_id: nil, owned_by_name: nil)
     Membership.where(user_id: id).delete_all
   end
+
+  def miner?
```

Comment:
Can we rename this method for something like `early_v2_user?` and parameterize the email through an env var? We can add Codeminer's email by default in .env.example.